common/namespace/namespace.go

```diff
@@ -322,6 +335,11 @@ func (ns *Namespace) CustomSearchAttributesMapper() CustomSearchAttributesMapper
 	return ns.customSearchAttributesMapper
 }
 
+func (ns *Namespace) NexusOutgoingService(name string) (spec *nexuspb.OutgoingServiceSpec, ok bool) {
+	spec, ok = ns.outgoingNexusServicesByName[name]
```

Comment:
Just curious why use the named return values instead of just `return ns.outgoingServicesByName[name]` ?

Reply:
Go doesn't infer that getting a value out of a map can return two values, I helped it out a bit.